extensions/typescript-language-features/src/commands/useTsgo.ts

```diff
@@ -0,0 +1,74 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { Command } from './commandManager';
+
+export class EnableTsgoCommand implements Command {
+	public readonly id = 'typescript.enableTsgo';
+
+	public async execute(): Promise<void> {
+		await updateTsgoSetting(true);
+	}
+}
+
+export class DisableTsgoCommand implements Command {
+	public readonly id = 'typescript.disableTsgo';
+
+	public async execute(): Promise<void> {
+		await updateTsgoSetting(false);
+	}
+}
+
+/**
+ * Updates the TypeScript Go setting and reloads extension host.
+ * @param enable Whether to enable or disable TypeScript Go
+ */
+async function updateTsgoSetting(enable: boolean): Promise<void> {
+	const tsgoExtension = vscode.extensions.getExtension('typescript.typescript-lsp');
+	// Error if the TypeScript Go extension is not installed with a button to open the GitHub repo
+	if (!tsgoExtension) {
+		return vscode.window.showErrorMessage(
+			'The TypeScript Go extension is not installed.',
```

Comment:
Even though it's mostly for dev, make sure to localize these UI strings: `vscode.t('The TypeScript Go extension is not installed.')`